tests/integration-tests/tests/update/test_update/test_sit_update_compute_instance/pcluster.config.disable_ht.ini

```diff
@@ -0,0 +1,25 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+
+scheduler = {{ scheduler }}
+master_instance_type = {{ instance }}
+compute_instance_type = {{ instance }}
+initial_queue_size = 0
+max_queue_size = 2
+vpc_settings = parallelcluster-vpc
+disable_hyperthreading = true
+
+
+[vpc parallelcluster-vpc]
```

Comment:
can you set use_public_ips to false so that we don't need to use one elastic ip for each test?

Reply:
Done